python/tskit/drawing.py

```diff
@@ -125,15 +149,13 @@ def rnd(x):
     if x == 0 or not math.isfinite(x):
         return x
     digits -= math.ceil(math.log10(abs(x)))
-    return round(x, digits)
-
-
-def add_text_in_group(dwg, elem, x, y, text, **kwargs):
-    """
-    Add the text to the elem within a group. This allows text rotations to work smoothly
-    """
-    grp = elem.add(dwg.g(transform=f"translate({rnd(x)}, {rnd(y)})"))
-    grp.add(dwg.text(text, **kwargs))
+    x = round(x, digits)
+    try:
```

Comment:
`if int(x) == x` would be a way to do this without the try except?

Reply:
Yep, if you prefer.